.github/ISSUE_TEMPLATES/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+
+## Describe the bug:
+A clear and concise description of what the bug is.
+
+---
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
```

Comment:
Its easy to link a picture on the web but screenshots are kind of a hassle because i don't think there's a way to directly import a picture into a markdown unless you upload it somewhere first? I don't know if there's a quick solution for this but might be something to think about.

Reply:
You can drag and drop in the github editor

Reply:
See: https://github.com/Mirascope/mirascope/issues/35